pkg/apis/networking/validation/validation_test.go

```diff
@@ -1093,6 +1289,64 @@ func TestValidateIngressUpdate(t *testing.T) {
 			},
 			expectedErrs: field.ErrorList{},
 		},
+		"no regex paths -> no regex paths": {
```

Comment:
should this be renamed, it actually looks like the `valid regex path -> valid regex path` case

also, do we have a tests that verify we tolerate:
* `invalid regex path -> invalid regex path`
* `invalid regex path -> valid regex path`

Reply:
Updated + added additional test cases, thanks!